src/components/collection/CollectionPlayerContainer.jsx

```diff
@@ -26,7 +26,7 @@ const CollectionPlayerContainer = ({
 
   // Helper fn to get segements
   const getSegments = useCallback((i) => collection.tracks[i].segments, [collection])
-  const getId = useCallback((i) => collection.tracks[i].id, [collection])
+  const getId = useCallback((i) => collection.tracks[i]?.id, [collection])
```

Comment:
what happens here? getId returns undefined and we check for it being so elsewhere? is that ok in the initial load scenario before we get a result back?

Reply:
Still errors in the console when trying to play etc, but the initial render loads:
![image](https://user-images.githubusercontent.com/3690498/143970389-ec7f471b-6d95-47af-9b6a-04275cf35817.png)

I can take a more holistic crack at it if we want to make an "empty" collection tile

Reply:
i think this is fine